centreon/src/Core/Host/Application/UseCase/PartialUpdateHost/HostMacroFactory.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace Core\Host\Application\UseCase\PartialUpdateHost;
+
+use Assert\AssertionFailedException;
+use Core\Macro\Domain\Model\Macro;
+
+final class HostMacroFactory
```

Comment:
I just noticed taht we know have 4 classes named `HostMacroFactory`.
I don't think that's a good idea, but anyway, forget it and just resolve this comment.
I couldn't make it as if I saw nothing, not your fault.